pipeline/terraform/stack/elastic_pipeline.tf

```diff
@@ -10,7 +10,7 @@ locals {
   #
   # Once we stop reindexing and make the pipeline live, we want it to be
   # highly available, because it's serving API traffic.
-  es_node_count = var.reindexing_state.scale_up_elastic_cluster ? 1 : 3
+  es_node_count = var.reindexing_state.scale_up_elastic_cluster ? 2 : 3
```

Comment:
This shouldn't cost us much extra, but it means this cluster will be continuously available during resizing.

This is useful if we ever need to change the size of a cluster after we start using it for live traffic; not doing this is what's scuppered the 2022-06-17 pipeline. I wanted to turn the cluster back up after it was live, but I couldn't do that without causing an outage.